scripts/python.sh

```diff
@@ -23,10 +22,9 @@ install-py-dev() {
     # Installing wheel will speed up installation of Python dependencies
     require wheel || pip install wheel
     echo "--> Installing Sentry (for development)"
-    pip_version=$(pip -V |  awk '{print $2}')
-    # Older versions of pip require SYSTEM_VERSION_COMPAT in Big Sur
-    # shellcheck disable=SC2072
-    if query_big_sur && [[ $pip_version < 20.3 ]]; then
+    # In Big Sur, versions of pip before 20.3 require SYSTEM_VERSION_COMPAT set
+    old_pip=$(python -c 'import sys; import pip; from pip._vendor.packaging.version import parse; print(1 if parse(pip.__version__) < parse("20.3") else 0)')
+    if query_big_sur && [[ $old_pip == 1 ]]; then
```

Comment:
You don't need to use print, I assume you did this because `$?` would be overwritten.

Just use exit and put python after the short circuiting && like this.

```suggestion
    if query_big_sur && python -c 'from sys import exit; import pip; from pip._vendor.packaging.version import parse; exit(1 if parse(pip.__version__) < parse("20.3") else 0)'; then
```